src/utils/tools.cpp

```diff
@@ -1673,3 +1673,22 @@ std::vector<std::string> split(const std::string &str) {
 	}
 	return tokens;
 }
+
+std::string getFormattedTime(uint32_t time) {
```

Comment:
Can't be removed? It is not used and I already added a function that formats the time.

Reply:
> Can't be removed? It is not used and I already added a function that formats the time.

We can maintain it for further usage.